llvm/test/CodeGen/LoongArch/soft-fp-to-int.ll

```diff
@@ -45,11 +45,8 @@ define i32 @fptosi_i32_double(double %X) nounwind {
 ;
 ; LA64-LABEL: fptosi_i32_double:
 ; LA64:       # %bb.0:
-; LA64-NEXT:    addi.d $sp, $sp, -16
-; LA64-NEXT:    st.d $ra, $sp, 8 # 8-byte Folded Spill
-; LA64-NEXT:    bl %plt(__fixdfsi)
-; LA64-NEXT:    ld.d $ra, $sp, 8 # 8-byte Folded Reload
-; LA64-NEXT:    addi.d $sp, $sp, 16
+; LA64-NEXT:    ftintrz.w.d $fa0, $fa0
+; LA64-NEXT:    movfr2gr.s $a0, $fa0
```

Comment:
Keeping soft-fp here.

Reply:
What I understand you to mean is that the purpose for this case should be kept, which is to test the soft-fp to int conversion.
Updated, added `--mattr=-f` to stay soft-fp.